data/fleet-commander-dbus.service.in

```diff
@@ -6,3 +6,8 @@ Type=dbus
 BusName=org.freedesktop.FleetCommander
```

Comment:
Have you actually tested this? This is still type=dbus... the point is not to make it a dbus service anymore but a normal one otherwise the clientdata might not get served unless someone calls the dbus service